src/pages/development/security/subresource-integrity.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Subresource Integrity
+description: Security feature that enables browsers to verify the resources they fetch.
+---
+
+# Subresource Integrity
+Subresource Integrity (SRI) is a security feature that enables browsers to verify that resources they fetch (for example, from a CDN) are delivered without unexpected manipulation. It works by allowing you to provide a cryptographic hash that a fetched resource must match.
+See [Subresource Integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) to learn more about SRI.
+
+## Application Support
+In order to adhere to PCI 4.0 requirements regarding verifying the integrity of scripts on payment pages, Adobe Commerce and Magento Open Source 2.4.7 and later include support for Subresource Integrity by providing integrity hashes for all javascript assets residing in the local filesystem. (This
+functionality is defined in the Magento_Csp module.)
+Please note that currently there is no SRI support for custom remote javascript resources.
+
+## Default Configuration
+The default SRI feature is implemented only on the payment pages for the admin and storefront areas. However, merchants can extend the default configuration to other pages.
+For example, to enable SRI on the customer account page in the storefront, create the file `<module-dir>/view/frontend/layout/customer_account_index.xml` with the following content
+```xml
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <head>
+        <link src="Magento_Csp::js/sri.js"/>
+    </head>
+    <body>
+        <referenceBlock name="head.additional">
+            <block class="Magento\Csp\Block\Sri\Hashes" name="customer.account.index.sri.hashes" template="Magento_Csp::sri/hashes.phtml"/>
+        </referenceBlock>
+    </body>
+</page>
+```
+
+## Subresource Integrity Hash Generation
+The Subresource Integrity hash generation process starts after [static content](https://experienceleague.adobe.com/en/docs/commerce-operations/configuration-guide/cli/static-view/static-view-file-deployment) for every package area is deployed.
+The [postprocessor](https://github.com/magento-commerce/magento2ce/tree/2.4-develop/app/code/Magento/Csp/Model/Deploy/Package/Processor/PostProcessor) class processes all javascript files by package and generates integrity hashes.
+The postprocessor class invokes the [SubresourceIntegrityCollector](https://github.com/magento-commerce/magento2ce/blob/2.4-develop/app/code/Magento/Csp/Model/SubresourceIntegrityCollector.php) class to accumulate files to save in cache once all packages are deployed.
```

Comment:
"class to accumulate files..." - "files" need to be replaced with "integrity hashes" or "hashes" since SubresourceIntegrityCollector does not accumulates files.